sp800-63b/sec8_security.md

```diff
@@ -31,6 +33,7 @@ This document assumes that the subscriber is not colluding with the attacker who
 | Eavesdropping | The authenticator secret or authenticator output is revealed to the attacker as the subscriber is authenticating. | Memorized secrets are obtained by watching keyboard entry. |
 | | | Memorized secrets or authenticator outputs are intercepted by keystroke logging software. |
 | | | A PIN is captured from PIN pad device. |
+| | An out of band secret is intercepted by the attacker by compromising the communication channel. | A one-time password is transmitted via unencrypted wifi and received by the attacker. |
```

Comment:
To avoid confusion with OTP devices, it is better to refer to a secret transmitted out of band as "out of band secret" rather than as a "one-time password". So suggest "A one-time password is transmitted" -> "An out of band secret is transmitted".